.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "Sming/gdbstub"]
```

Comment:
We don't currently use sub-modules, this is something that's under discussion though... see #69.